utils/spacewalk-common-channels.ini

```diff
@@ -2166,3 +2166,111 @@ repo_type = deb
 checksum = sha256
 base_channels = astralinux-orel-pool-amd64
 repo_url = https://download.astralinux.ru/astra/stable/orel/repository/dists/orel/non-free/binary-amd64/
+
+[alibaba-2]
+archs    = x86_64
+name     = Alibaba Cloud Linux (Aliyun Linux) release 2 (%(arch)s)
+checksum = sha256
+gpgkey_url = https://mirrors.aliyun.com/alinux/RPM-GPG-KEY-ALIYUN
+gpgkey_id  = 873141A8
+gpgkey_fingerprint = EFD7 52E7 E232 ED87 12E7  635C EB80 1C41 8731 41A8
+repo_url = https://mirrors.aliyun.com/alinux/2/os/%(arch)s/
+dist_map_release = 2
+
+[alibaba-2-exp]
+label    = %(base_channel)s-exp
+archs    = x86_64
+name     = Development Packages for Alibaba Cloud Linux (Aliyun Linux) release 2 (%(arch)s)
+base_channels = alibaba-2-%(arch)s
+repo_url = https://mirrors.aliyun.com/alinux/2/exp/%(arch)s/
+
+[alibaba-2-extras]
+label    = %(base_channel)s-extras
+archs    = x86_64
+name     = Extras for Alibaba Cloud Linux (Aliyun Linux) release 2 (%(arch)s)
+base_channels = alibaba-2-%(arch)s
+repo_url = https://mirrors.aliyun.com/alinux/2/extras/%(arch)s/
+
+[alibaba-2-kernels]
+label    = %(base_channel)s-kernels
+archs    = x86_64
+name     = Additional kernels for Alibaba Cloud Linux (Aliyun Linux) release 2 (%(arch)s)
+base_channels = alibaba-2-%(arch)s
+repo_url = https://mirrors.aliyun.com/alinux/2/kernels/%(arch)s/
+
+[alibaba-2-plus]
+label    = %(base_channel)s-plus
+archs    = x86_64
+name     = Alibaba Cloud Linux (Aliyun Linux) release 2 Plus (%(arch)s)
+base_channels = alibaba-2-%(arch)s
+repo_url = https://mirrors.aliyun.com/alinux/2/plus/%(arch)s/
+
+[alibaba-2-updates]
+label    = %(base_channel)s-updates
+archs    = x86_64
+name     = Alibaba Cloud Linux (Aliyun Linux) release 2 Updates (%(arch)s)
+base_channels = alibaba-2-%(arch)s
+repo_url = https://mirrors.aliyun.com/alinux/2/updates/%(arch)s/
+
+[alibaba-2-aarch64]
+archs    = aarch64
+name     = Alibaba Cloud Linux (Aliyun Linux) release 2 (%(arch)s)
+checksum = sha256
+gpgkey_url = https://mirrors.aliyun.com/alinux/RPM-GPG-KEY-ALIYUN
+gpgkey_id  = 873141A8
+gpgkey_fingerprint = EFD7 52E7 E232 ED87 12E7  635C EB80 1C41 8731 41A8
+repo_url = https://mirrors.aliyun.com/alinux/2/os-arm/%(arch)s/
```

Comment:
Damned non standard URLs. Same problem for openSUSE. Otherwise we could define a single set of channels for both `x86_64` and `aarch64`